ksql-rest-model/src/main/java/io/confluent/ksql/rest/DefaultErrorsImpl.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest;
+
+import javax.ws.rs.core.Response;
+
+public class DefaultErrorsImpl implements Errors {
+
+  @Override
+  public Response accessDeniedFromKafkaResponse(final Throwable t) {
```

Comment:
We shouldn't be handling `Throwable` here, only exceptions.  I know this is copied from existing code, but given this is becoming a public API we need to think about such things.

```suggestion
  public Response accessDeniedFromKafkaResponse(final Exception e) {
```